contracts/staking/test/pools_test.ts

```diff
@@ -47,12 +46,23 @@ blockchainTests('Staking Pool Management', env => {
             const operatorShare = 39;
             const poolOperator = new PoolOperatorActor(operatorAddress, stakingWrapper);
             // create pool
-            const poolId = await poolOperator.createStakingPoolAsync(operatorShare);
+            const poolId = await poolOperator.createStakingPoolAsync(operatorShare, false);
             expect(poolId).to.be.equal(stakingConstants.INITIAL_POOL_ID);
             // check that the next pool id was incremented
-            const expectedNextPoolId = '0x0000000000000000000000000000000200000000000000000000000000000000';
             const nextPoolId = await stakingWrapper.getNextStakingPoolIdAsync();
-            expect(nextPoolId).to.be.equal(expectedNextPoolId);
+            expect(nextPoolId).to.be.equal(stakingConstants.SECOND_POOL_ID);
+        });
+        it('Should successfully create a pool and add owner as a maker', async () => {
+            // test parameters
+            const operatorAddress = users[0];
+            const operatorShare = 39;
+            const poolOperator = new PoolOperatorActor(operatorAddress, stakingWrapper);
+            // create pool
+            const poolId = await poolOperator.createStakingPoolAsync(operatorShare, true);
+            expect(poolId).to.be.equal(stakingConstants.INITIAL_POOL_ID);
+            // check that the next pool id was incremented
+            const nextPoolId = await stakingWrapper.getNextStakingPoolIdAsync();
```

Comment:
Let's throw this check into the `PoolOperatorActor` class.

Reply:
o wait actually the reason this is here is because we're checking the poolId against a string constant, so to do the check in `createStakingPoolAsync` we'd need to increment the pool id in TypeScript, which is probably not worth